webapp/Makefile

```diff
@@ -8,7 +8,12 @@ CI ?= false
 # please see optipng-bin Linux arm64 support issue (https://github.com/imagemin/optipng-bin/issues/118) for details:
 ifeq ($(shell uname)/$(shell uname -m),Linux/aarch64)
   LINUX_ARM64 = true
-  CPPFLAGS = "-DPNG_ARM_NEON_OPT=0"
+  CPPFLAGS += " -DPNG_ARM_NEON_OPT=0"
```

Comment:
For my own curiosity, is changing this to a `+=` instead of an `=` because you're passing other flags in? Or is it just a good practice?

I guess there's a chance that this would cause additional flags to be passed in from the environment on ARM64, but since CPPFLAGS is just passed in directly on other architectures, that's not likely to cause an issue.

Reply:
The latter.  Always want to respect the user's environment.